### ISSUE TITLE
Sync vs async comparison

### ISSUE THREAD

**Issue by inikolaev** [larissaleite/concurrency-and-async#1]
I would like to point out a mistake in the sync implementation, which I think makes comparison with async not very fair.

The code uses `requests.get` directly, but the problem is that it doesn't use connection pooling and a new connection is created on each request, whereas async version re-uses `AsyncClient` which maintains connection pool.

Instead it's better to create a session and then use it in-place of `requests`:

```python
session = requests.Session()
session.get('https://www.google.com')
```

**Comment by larissaleite** [comment]
Good point, I think I was recreating the async client before, similarly to the sync implementation, but changed after a few iterations. Will compare it using session 👍
